string uri
string tf_prefix
float32 roll_trim
float32 pitch_trim
bool enable_logging
---
